doc/release_notes.qbk

```diff
@@ -19,6 +19,20 @@
 
 [section:release_notes Release Notes]
 
+[/=================]
+[heading Boost 1.87]
+[/=================]
```

Comment:
If it does not make it to 1.88, we'll update

Reply:
✅

Reply:
@vissarion , I updated to 1.88 yesterday but the branch is open again for bug fixes.
So this one can probably make it.
One other PR will follow today, I'll make a separate commit and add that one also there, and change the date.

Also, I noticed yesterday some bugs that were not listed, I'll add them also there (I remember they were not merged in August so that is also 1.87).

Reply:
Thanks I will take care of the release, i.e. I will check if something is missing from the release notes of 1.87 and create a PR. We have roughly two more weeks.